pipelines/rpn_signature.py

```diff
@@ -203,12 +199,9 @@ def rpn(wf, **kwargs):
 
     wf.write_graph('rpn-signature.png')
 
-run_args = {
-    'plugin':'MultiProc',
-    'plugin_args':{'n_procs':4,'memory_gb':5}
-}
-
 
-if __name__ == '__main__':
-    print("Starting RPN-signature...")
-    rpn_wf = rpn('rpn', bids_dir=input_dir, subjects=['001'], run_args=run_args)
+rpn_app = BidsApp(
```

Comment:
where can we provoide runtime params like "MultiProc", N_procs, etc..?

Reply:
oh is it run_args?

Reply:
can we than create a BIDSapp where run_args can be set from the command line?

Reply:
Currently, it is not possible to pass run_args via the cli, only via the BidsApp constructor. But I can gladly change that.

If run_args is specified via cli as well as via the constructor, we decide to use the run_args specified by the cli, right?